en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -33,4 +33,46 @@ To see all possible command line arguments, check out:
 
 ```sh
 test/mavsdk_tests/mavsdk_test_runner.py -h
+
+usage: mavsdk_test_runner.py [-h] [--log-dir LOG_DIR] [--speed-factor SPEED_FACTOR] [--iterations ITERATIONS] [--abort-early] [--gui] [--model MODEL]
+                             [--case CASE] [--debugger DEBUGGER] [--verbose]
+                             config_file
+
+positional arguments:
+  config_file           JSON config file to use
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --log-dir LOG_DIR     Directory for log files
+  --speed-factor SPEED_FACTOR
+                        How fast to run the simulation
+  --iterations ITERATIONS
+                        How often to run all tests
+  --abort-early         Abort on first unsuccessful test
+  --gui                 Display gzclient with simulation
+  --model MODEL         Only run tests for one model
+  --case CASE           Only run tests for one case
+  --debugger DEBUGGER   valgrind callgrind gdb lldb
+  --verbose             Enable more verbose output
 ```
+
+## Notes on implementation
+
+
+- The tests are invoked from the test runner script [mavsdk_test_runner.py](https://github.com/PX4/Firmware/blob/master/test/mavsdk_tests/mavsdk_test_runner.py), which is written in Python.
+  This runner also starts `px4` as well as Gazebo for SITL tests, and collects the logs of these processes.
+- The test runner is a C++ binary
+  It contains:
+  - The [main](https://github.com/PX4/Firmware/blob/master/test/mavsdk_tests/test_main.cpp) function to parse the arguments.
+  - An abstraction around MAVSDK called [autopilot_tester](https://github.com/PX4/Firmware/blob/941a917d38d4cccb87ab6d342b0b1cee2cd03e80/test/mavsdk_tests/autopilot_tester.h).
```

Comment:
do you really want this specific blob or should we link to master?

Reply:
That was by mistake, thanks!